### ISSUE TITLE
How to update this project for the latest version of the Latin IME? 

### ISSUE THREAD

**Issue by vedantroy** [ihrupin/LatinIME-gradle-module#1]
I have been trying to turn the AOSP Latin IME into a Gradle module. Recently, I found this repository. Unfortunately, this repository is not updated for the latest version of the Latin IME.

I would like to update this project to the latest version of the Latin IME. So far, I have been able to update the Java and XML files, but I am stuck on how to update the C++ files. The Latin IME is now built with the Soong build system, which uses "android.bp" files instead of "android.mk" files. 

How do I update this project to support the latest version of the Latin IME?

In summary--while I have figured out how to create a Gradle module with the Latin IME's Java and XML files, I do not know how to compile the native dependencies that the Latin IME requires.

**Comment by nift4** [comment]
Try https://github.com/iwo/LatinIME-Android-Studio
